src/OpenTelemetry.Api/Context/Propagation/TraceContextPropagator.cs

```diff
@@ -38,6 +38,10 @@ public class TraceContextPropagator : TextMapPropagator
         private static readonly int VersionAndTraceIdAndSpanIdLength = "00-0af7651916cd43dd8448eb211c80319c-00f067aa0ba902b7-".Length;
         private static readonly int OptionsLength = "00".Length;
         private static readonly int TraceparentLengthV0 = "00-0af7651916cd43dd8448eb211c80319c-00f067aa0ba902b7-00".Length;
+        private static readonly int TraceStateKeyMaxLength = 256;
+        private static readonly int TraceStateKeyTenentMaxLength = 241;
```

Comment:
I'm not sure, but I think it's `Tenant`

Reply:
If we have a spec showing where these magic numbers come from, would be nice to add as a comment.

Reply:
> 
> 
> I'm not sure, but I think it's `Tenant`

Good catch! I've updated all. (I mixed it with "Tenet". XD)

As for the magic numbers, I had links in line in the methods, but I also added here for readability.

Reply:
perfect! awesome!